dataland-frontend/src/components/general/AlignedActivitiesDataTable.vue

```diff
@@ -197,6 +222,20 @@ export default defineComponent({
     ];
   },
   methods: {
+    /**
+     * @param value the number to format as a percentage
+     * @returns the new formatted string
+     */
+    valueFormatterForPercentage(value: number | undefined): string {
+      return typeof value !== "undefined" ? `${value}%` : "";
+    },
+    /**
+     * @param value the YesNo value to format as Yes, No or blank
+     * @returns the new formatted string
+     */
+    valueFormatterForYesNo(value: YesNo): string {
+      return typeof value !== "undefined" ? value : "";
```

Comment:
Same as `valueFormatterForPercentage`

Reply:
The AlignedActivitiesDataTable.vue is being reworked in DALA-2307. Re-Check if issues persist after merge (I.e., do not work on this issue until DALA-2307 has been merged)

Reply:
I will look into this

Reply:
Is removed

Reply:
valueFormatterForPercentage also removed